<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>621</width>
     <height>461</height>
    </size>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="setTab">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Drag And Drop</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="OnRadiobutton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>On</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="OffRadioButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Monday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wednsday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thursday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Firday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Saturday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sunday</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>100</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>First Day of the Week</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="dayTab">
    <attribute name="title">
     <string>Day</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="weekTab">
    <attribute name="title">
     <string>Week</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="monthTab">
    <attribute name="title">
     <string>Month</string>
    </attribute>
    <widget class="myCalendar" name="calendar">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>431</width>
       <height>401</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <underline>false</underline>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-color: rgba(128, 0, 255, 151);
</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
    </widget>
    <widget class="QLabel" name="eventLabel">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>0</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">eventLabel{
background-colour:rgb(255, 255, 255),
colour: rgba(0, 128, 255, 147)

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>   MyEvent</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>181</width>
       <height>311</height>
      </rect>
     </property>
     <widget class="DisplayListWidget" name="displayList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>91</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="yearTab">
    <attribute name="title">
     <string>Year</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myCalendar</class>
   <extends>QCalendarWidget</extends>
   <header location="global">mycalendar.h</header>
  </customwidget>
  <customwidget>
   <class>DisplayListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">displaylistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
